jenkins/spark-tests.sh

```diff
@@ -304,6 +304,11 @@ if [[ $TEST_MODE == "DEFAULT" ]]; then
   PYSP_TEST_spark_shuffle_manager=com.nvidia.spark.rapids.${SHUFFLE_SPARK_SHIM}.RapidsShuffleManager \
     ./run_pyspark_from_build.sh
 
+  SPARK_SHELL_SMOKE_TEST=1 \
```

Comment:
There is a limitation in Spark that it currently does not support classifiers https://issues.apache.org/jira/browse/SPARK-20075

It only works because our default artifact is a cuda11 jar, Can not work on arm64 classifier

https://github.com/NVIDIA/spark-rapids/issues/10160#issuecomment-1881527250